contracts/Ovm_SpokePool.sol

```diff
@@ -43,6 +44,29 @@ contract Ovm_SpokePool is SpokePool {
     // Address of SNX ERC20
     address private constant SNX = 0x8700dAec35aF8Ff88c16BdF0418774CB3D7599B4;
 
+    /**
+     * @notice Domain identifier used for Circle's CCTP bridge to L1.
+     * @dev This identifier is assigned by Circle and is not related to a chain ID.
+     * @dev Official domain list can be found here: https://developers.circle.com/stablecoins/docs/supported-domains
+     */
+    uint32 public constant l1CircleDomainId = 0;
+    /**
+     * @notice The official USDC contract address on this chain.
+     * @dev Posted officially here: https://developers.circle.com/stablecoins/docs/usdc-on-main-networks
+     */
+    IERC20 public l2Usdc;
```

Comment:
if we're adding a new storage variable to a base contract like this one we need to decrement this contract's `__gap` variable. This contract is a base contract for Optimism/Base/Boba_SpokePool's. Decrement the `__gap` for each 32 byte slot used up: https://docs.openzeppelin.com/upgrades-plugins/1.x/writing-upgradeable#storage-gaps

Reply:
I've decremented the `__gap` by 2 slots to include space for the `CircleCCTPAdapter`'s `recipientCircleDomainId`, `usdcToken`, and `cctpTokenMessenger`. 

This is because: `recipientCircleDomainId` is 4 bytes, and `usdcToken` & `cctpTokenMessenger` should be 20 bytes each. So in total, 44 bytes fit in 2 slots.